test/factories/articles.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "active_model"
```

Comment:
Note: Rails expects callers to do a top-level require of the library you're using, so it doesn't have to insert as many requires into each individual piece. So this satisfies that contract.

Reply:
Should we leave the `require`s on the next 2 lines then?